src/api/api-config.js

```diff
@@ -0,0 +1,26 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const fetchMissions = createAsyncThunk(
+  'missions / fetchMissions',
+  async () => {
+    try {
+      const response = await axios.get('https://api.spacexdata.com/v3/missions');
+      return response.data;
+    } catch (error) {
+      throw new Error('Failed to fetch missions.');
+    }
+  },
+);
+
+export const fetchRockets = createAsyncThunk(
+  'Rockets / fetchRockets',
+  async () => {
+    try {
+      const response = await axios.get('https://api.spacexdata.com/v3/rockets');
+      return response.data;
+    } catch (error) {
+      throw new Error('Failed to fetch rocketss.');
+    }
+  },
+);
```

Comment:
- Optional
I would recommend removing this file since it is not being used anywhere. Remember it is super important to have a clean and professional repo. Having useless/unnecessary files ofter creates a mess and makes it difficult to navigate the project.